src/main/java/edu/harvard/iq/dataverse/ingest/tabulardata/impl/plugins/sav/SAVFileReader.java

```diff
@@ -29,7 +29,7 @@
 import java.io.UnsupportedEncodingException;
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
-
+import java.nio.charset.StandardCharsets;
```

Comment:
Would it make sense to also fix the other lines that use string representations of char sets?

282: private String defaultCharSet = "ISO-8859-1";
private String defaultCharSet = StandardCharsets.ISO_8859_1.name();

700: defaultCharSet = "UTF-8";

Reply:
Looking at where `defaultCharSet` is read (only in `new String(.., defaultCharSet)`, it would make sense to make it an actual `Charset`. How does that sound?

Reply:
Using StandardCharsets instead of the string name is a good idea.

Reply:
I don't know exactly how this code interacts with the UI and APIs, which seem to allow users to choose from a list of charsets for certain files. Can we make this a separate issue?

Reply:
Ok